### PR TITLE
Co040424 rep status authenticated

### PR DESCRIPTION
**Pull request by cosu419**
## Summary
With this PR, the Representative Status widget is now calling the `/representation_management/v0/power_of_attorney` endpoint. The "authenticated (individual)" state is now in place. 

`va-icon` components continue to display locally, but not in staging. 

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/77602

## Screenshot
Using hardcoded rep data:
image

## Testing done
Added simple axe check for authenticated state.

### CODE REVIEW

File: src/applications/representative-search/containers/SearchPage.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ import appendQuery from 'append-query';
 import { browserHistory } from 'react-router';
 import repStatusLoader from 'applications/static-pages/representative-status';
 import environment from '@department-of-veterans-affairs/platform-utilities/environment';
-import { useFeatureToggle } from '~/platform/utilities/feature-toggles/useFeatureToggle';
+// import { useFeatureToggle } from '~/platform/utilities/feature-toggles/useFeatureToggle';
```
Comment (jvcAdHoc) [inline src/applications/representative-search/containers/SearchPage.jsx:16]: Did you want to keep some of these lines commented out?

Comment (cosu419) [reply src/applications/representative-search/containers/SearchPage.jsx]: Removed. Find a Rep won't load on older versions of Safari, and I have a hunch it's related to an issue with loading feature toggles. So I may bring the toggle back, but I'm good with deleting everything for now.

File: src/applications/static-pages/representative-status/components/App/States/Auth.jsx [hunk 2]
```diff
@@ -1,38 +1,208 @@
...
-            </div>
-            <div className="auth-no-rep-text">
-              <DynamicHeader className="auth-no-rep-header">
-                You don’t have an accredited representative
-              </DynamicHeader>
-              <div className="auth-no-rep-body">
+export const Auth = ({ DynamicHeader, DynamicSubheader }) => {
+  const [representative, setRepresentative] = useState({});
+
+  const fetchRepStatus = async () => {
```
Comment (jvcAdHoc) [inline src/applications/static-pages/representative-status/components/App/States/Auth.jsx:9]: What are your thoughts on moving the network call outside of the component? I think the component's responsibility should only be to render the information for a provided representative prop or render the empty state

File: src/applications/static-pages/representative-status/utilities/phoneNumbers.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+export const parsePhoneNumber = phone => {
+  if (!phone) {
+    return { contact: null, extension: null };
+  }
+  let sanitizedNumber = phone
+    .replace(/[()\s]/g, '') // remove parentheses
+    .replace(/(?<=.)([+.*])/g, '-'); // replace .*+ symbols being used as dashes
+
+  // return null for non-US country codes
+  if (sanitizedNumber.match(/\+(\d+)[^\d1]/g)) {
```
Comment (jvcAdHoc) [inline src/applications/static-pages/representative-status/utilities/phoneNumbers.js:24]: if `.replace(/(?<=.)([+.*])/g, '-'); // replace .*+ symbols being used as dashes` works, then how would this condition looking for `+` be true?

**Review by powellkerry — DISMISSED** [review]

File: src/applications/static-pages/representative-status/utilities/phoneNumbers.js (hunk 3 above)
Comment (cosu419) [reply src/applications/static-pages/representative-status/utilities/phoneNumbers.js:24]: The way I'm determining whether a `+` is "being used as a dash" is if it follows another character. So a number formatted like `+1+555+123+4567` would be converted to `+1-555-123-4567`.

Comment (jvcAdHoc) [reply src/applications/static-pages/representative-status/utilities/phoneNumbers.js:24]: I missed that, thanks!

**Review by jvcAdHoc — APPROVED** [review]

**Review by powellkerry — APPROVED** [review]

**Pull request merged**